src/middlewared/middlewared/plugins/disk.py

```diff
@@ -602,7 +601,7 @@ async def multipath_sync(self):
 
         reserved = await self.get_reserved()
 
-        devlist = await camcontrol_list()
+        devlist = []
```

Comment:
`devlist` is expected to be a dictionary which would result in an error at L620 below